pytket/extensions/qiskit/backends/aer.py

```diff
@@ -237,8 +237,14 @@ def process_circuits(
         circuits: Sequence[Circuit],
         n_shots: Union[None, int, Sequence[Optional[int]]] = None,
         valid_check: bool = True,
-        **kwargs: KwargTypes,
+        **kwargs: Union[bool, int, float, str, None],
     ) -> List[ResultHandle]:
+        """
+        See :py:meth:`pytket.backends.Backend.process_circuits`.
+        Supported kwargs: `seed`, `postprocess`, `seed_auto_increase`.
+        seed_auto_increase=True will automatically increase the seed by one for the
```

Comment:
I think this should be automatic (definitely the default behaviour, but I don't see a reasonable use case for _not_ incrementing the seed, which introduces biases in the results, so no need to add a new kwarg).

Reply:
I did not wanted to do this by default because that would stop old runs from being reproducible, if that is not a concern I am happy to remove the parameter and always do the auto inc. The use case for the parameter I see is being compatible with reproducing old runs.

Reply:
At least they will still be reproducible if the same version numbers (of pytket-qiskit, qiskit, etc) are used.